ambry-server/src/main/java/com/github/ambry/server/AmbryServer.java

```diff
@@ -63,6 +64,7 @@
 import com.github.ambry.rest.RestRequestResponseHandlerFactory;
 import com.github.ambry.rest.RestRequestService;
 import com.github.ambry.rest.StorageServerNettyFactory;
+import com.github.ambry.router.Callback;
```

Comment:
remove this and make sure all the Callback related imports are updated in this PR